manual/cops_rails.md

```diff
@@ -226,6 +226,82 @@ NilOrEmpty | `true` | Boolean
 NotPresent | `true` | Boolean
 UnlessPresent | `true` | Boolean
 
+## Rails/BulkChangeTable
+
+Enabled by default | Supports autocorrection
+--- | ---
+Enabled | No
+
+This Cop checks whether alter queries are combinable.
+If combinable queries are detected, it suggests to you
+to use `change_table` with `bulk: true` instead.
+When use this method, make combinable alter queries
```

Comment:
This sentence doesn't seem to make sense to me

Reply:
Sorry. This is a bit buried in a merged PR, so I've made an issue of it https://github.com/rubocop-hq/rubocop/issues/6158